website/docs/docs/build/metricflow-cli.md

```diff
@@ -0,0 +1,365 @@
+---
+title: MetricFlow CLI
+id: metricflow-cli
+description: "Query metrics and metadata in your dbt project with the metricflow cli"
+sidebar_label: "MetricFlow CLI commands"
+tags: [Metrics, Semantic Layer]
+---
+
+Once you define metrics in your dbt project, you can query metrics, dimensions, dimension values, and validate your configs using the MetricFlow command line (CLI).
+
+# Installation
+
+You can install the [MetricFlow CLI](https://github.com/dbt-labs/metricflow#getting-started) from [PyPI](https://pypi.org/project/dbt-metricflow/). You need to use `pip` to install the MetricFlow CLI on Windows or Linux operating systems:
+
+1. Create or activate your virtual environment.`python -m venv venv`
+2. Run `pip install dbt-metricflow`
+
+The MetricFlow CLI is compatible with Python versions 3.8, 3.9, 3.10 and 3.11
+
+# CLI commands
+
```

Comment:
<!-- this section may best live under references -> commands. after speaking with Leona, for the purposes of the SL launch, it's better for the user to have the metricflow docs in one place while they navigate. We can re visit this at a later point. -->